lib/logstash/plugin_mixins/rabbitmq_connection.rb

```diff
@@ -40,9 +40,6 @@ def setup_rabbitmq_connection_config
         # Specify TLS version if using TLS e.g "TLSv1.2"
         config :ssl, :validate => :string
 
-        # Validate SSL certificate
-        config :verify_ssl, :validate => :boolean, :default => false
```

Comment:
even if this did nothing, should we go thorugh the deprecated-then-obsolete process to help users (who have this setting) not be confused at its disappearance?

We could keep it and mark it `:deprecated => "This setting never actually did anything. <insert recommendation here>"` -- thoughts?